src/NServiceBus.Persistence.AzureStorage/SagaPersisters/AzureStorageSagaPersistence.cs

```diff
@@ -12,8 +11,10 @@ internal AzureStorageSagaPersistence()
             DependsOn<Features.Sagas>();
             Defaults(s =>
             {
-                var defaultConnectionString = ConfigurationManager.AppSettings["NServiceBus/Persistence"];
+#if NET452
```

Comment:
Do we need to handle .netstadard2.0 target case somehow?

Reply:
IMO a lib reading from AppSettings is an antipattern. that should be a hosting code concern

Reply:
@SeanFeldman can you raise an issue about this?

Reply:
Issue to do what @danielmarbach?

Reply:
@SeanFeldman 

> Sean: Do we need to handle .netstadard2.0 target case somehow?
> Simon: IMO a lib reading from AppSettings is an antipattern. that should be a hosting code concern

Sounds like something that should be discussed in an issue?

Reply:
@danielmarbach raised an [issue](https://github.com/Particular/PlatformDevelopment/issues/1455)